src/server/server.ts

```diff
@@ -99,7 +99,7 @@ setInterval(() => {
                     // Delete all directories older than three hours and all files older than three hours that are not in map
                     if ((now - stat.mtime.getMilliseconds()) > THREE_HOUR_MS) {
                         if (stat.isDirectory()) {
-                            rimraf(absolutePath, () => {});
+                            rimraf(absolutePath, () => { });
```

Comment:
can you just pass in null as the parameter or do you need to pass in a no-op function?